src/Core/src/Handlers/Entry/EntryHandler.Android.cs

```diff
@@ -68,8 +64,10 @@ protected override void DisconnectHandler(AppCompatEditText platformView)
 			_set = false;
 		}
 
-		public static void MapBackground(IEntryHandler handler, IEntry entry) =>
+		public static void MapBackground(IEntryHandler handler, IEntry entry)
```

Comment:
Am I missing something here? The previous conversation was resolved but this change wasn't removed.

Reply:
Initially in the PR we modified this extension method passing the `defaultBackgroundDrawable` as a parameter. Now and after the last changes, the parameters of this method are exactly the same.